packages/interfaces/test/chain.test.ts

```diff
@@ -157,8 +164,7 @@ function runTestSuite<T extends ChainImplementation<S, any>, S>({
 			timestamp: 10000000,
 		}
 		const out = await chainImplementation.signDelegatedAction(delegatedSigner, actionPayload)
-		t.notThrows(async () => {
-			await chainImplementation.verifyAction(out)
-		})
+		await chainImplementation.verifyAction(out)
+		t.pass()
```

Comment:
These changes don't change whether CI passes/fails, but if an exception is raised we should get a proper `[fail]` message rather than just displaying an error that says an exception has not been caught.